src/utils/SparseGrid.h

```diff
@@ -79,10 +79,21 @@ class SparseGrid
      * \param[in] radius The search radius.
      * \param[in] process_func Processes each element.  process_func(elem) is
      *    called for each element in the cell. Processing stops if function returns false.
+     * \return Whether we need to continue procesing after this function
      */
-    void processNearby(const Point &query_pt, coord_t radius,
+    bool processNearby(const Point &query_pt, coord_t radius,
                        const std::function<bool (const ElemT&)>& process_func) const;
 
+    /*! \brief Process elements from all cells.
+     *
+     * Processes all elements.
+     *\
+     * \param[in] process_func Processes each element.  process_func(elem) is
+     *    called for each element in the cells. Processing stops if function returns false.
+     * \return Whether we need to continue procesing after this function
+     */
+    bool processAll(const std::function<bool(const Elem& elem)>& process_elem_func) const;
```

Comment:
This new `processAll` function is unused. And that also is the only reason why the rest of the `processX` functions have a return type now. Why was this necessary?

Reply:
`PolygonUtils::findConnection` uses the return value of `processLine`

I found out that it makes a lot of sense to return whether to continue for all the processing functions equally.

I used `processAll` in debugging along the way and thought that it's a shame to throw the code changes away.

Perhaps `processAll` itself is not really needed.
Do you want me to remove it or keep it as a nice utility function for future use?

Reply:
I'd prefer to have it removed. Less code to maintain, and I don't see it being used in the near future.
(If this is the only issue I find now I'll continue testing though without waiting for those code changes.)

Reply:
Ok. I'll remove it.

Reply:
b3f200e59790b5e89756b42575b065e99fb7bba6